packages/main/src/plugin/kubernetes/contexts-manager.ts

```diff
@@ -386,7 +386,13 @@ export class ContextsManager {
           sendGeneral: true,
           currentContext: this.kubeConfig.currentContext,
           resources: { pods: true },
-          update: state => state.resources.pods.push(obj),
+          update: state => {
+            if (state.resources.pods.filter(o => o.metadata?.uid !== obj.metadata?.uid).length) {
+              console.debug(`==> resource ${obj.metadata?.name} already added`);
```

Comment:
I would for consistency, not use any prefix like ==>
Also if it happens, the trace should contain more contextual information like what is the kubeConfig.currentContext, etc

also about the `filter(predicate).length`, should it just use the `some(predicate)` operation